src/Util/ProjectionUtil/ProjectionUtil.js

```diff
@@ -0,0 +1,60 @@
+import proj4 from 'proj4';
+import OlProjection from 'ol/proj';
+
+/**
+ * Helper class for ol/proj4 projection handling.
+ *
+ * @class
+ */
+export class ProjectionUtil {
+
+  /**
+   * Returns an object of (custom) proj4 definitions.
+   *
+   * @return {Object} The (custom) proj4 definition strings.
+   */
+  static proj4CrsDefinitions() {
+    return {
+      'EPSG:25832': '+proj=utm +zone=32 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs'
```

Comment:
This is not in in the proj database? Or am I misunderstanding stuff here

Reply:
Is this because we get back the  urn-whatever identifiers? is there some way to look these up as well?